scholly/base/src/layouts/layout1/components/centred.css

```diff
@@ -0,0 +1,192 @@
+/** This file centralize the styling of client website */
```

Comment:
nit: can we rename all instances of `centred.css` to `centered.css`? I prefer the latter spelling and ensures spelling consistency across our app.

Reply:
Yep, no worries.